modal/queue.py

```diff
@@ -23,11 +23,31 @@ class _Queue(_Object, type_prefix="qu"):
 
     The queue can contain any object serializable by `cloudpickle`, including Modal objects.
 
-    **Lifetime of a queue and its contents**
+    By default, the `Queue` object acts as a single FIFO queue which supports puts and gets (blocking and non-blocking).
 
-    A `Queue`'s lifetime matches the lifetime of the app it's attached to, but the contents expire after 30 days.
-    Because of this, `Queues`s are best used for communication between active functions and not relied on for
-    persistent storage. On app completion or after stopping an app any associated `Queue` objects are cleaned up.
+    **Queue partitions (beta)**
+
+    Specifying partition keys gives access to other independent FIFO partitions within the same `Queue` object.
+    Across any two partitions, puts and gets are completely independent. For example, a put in one partition does not affect
```

Comment:
Super super nit but some of these lines are very long. I guess there's no way to get ruff to auto-wrap docstrings?